src/sentry_plugins/pushover/plugin.py

```diff
@@ -30,6 +35,7 @@ def get_config(self, **kwargs):
         )
 
         apikey_field.update({'name': 'apikey', 'label': 'API Key'})
+
         return [
             userkey_field, apikey_field, {
                 'name':
```

Comment:
No idea why this was originally formatted this way. Care to correct it? Let's not continue that trend. So, if you have:
```python
{
'name':
'retry',
'label':
'Retry',
'type':
'number',
'required':                
False,
}
```
It should actually look like:
```python
{
'name': 'retry',
'label': 'Retry',
'type': 'number',
'required': False,
}
```

Typically, in larger codebases there's a code formatter. I don't remember if there's in this repository.

Reply:
The old code formatter did this.

https://github.com/getsentry/sentry-plugins/commit/b54b6dcfc1300fb8d9087a6f849a15a60e486259

Reply:
Man.... that's really weird. Looks like it made a mistake. I suppose because the `choices` thing.

Reply:
I mean, I'm not going to defend it. :) Just pointing out that a human did not write code that looked like that.